hail/src/main/scala/is/hail/io/RowStore.scala

```diff
@@ -943,6 +943,216 @@ object EmitPackDecoder {
   }
 }
 
+object NativeDecoder {
```

Comment:
I think the diff from this file is probably left over from a bad rebase, right?

Reply:
Yep, fixing...